build.sbt

```diff
@@ -486,7 +486,7 @@ val zio = Seq(
 val bcpkixJdk15Version      = "1.70"
 val declineVersion          = "2.4.1"
 val directoryWatcherVersion = "0.18.0"
-val flatbuffersVersion      = "1.12.0"
+val flatbuffersVersion      = "24.3.25"
```

Comment:
Looks like a pretty recent version of `flatbuffers`. What will be the setup on my Ubuntu 22.04? Will I have to upgrade to new Ubuntu?

Reply:
For `./run`, it will automatically download the binary, so no additional setup is needed. 
If you avoid run, you'll have to provide the `flatc` binary on your own. The reliable way should be [downloading release asset](https://github.com/google/flatbuffers/releases/tag/v24.3.25) but if you have a package manager that happens to support that particular version, you can use it.